### DISCUSSION TITLE
Raptor Source Files

### DISCUSSION THREAD

**Discussion by spacek531** [OpenRCT2/OpenGraphics#28] [Official Objects]
I'm not sure where this should go specifically, but these are all the source files I used to make the RMC Raptor train.
[Raptor Project Files.zip](https://github.com/OpenRCT2/OpenGraphics/files/7030013/Raptor.Project.Files.zip)

As an official OpenRCT2 custom object, its source files should be somewhere.

**Comment by oli414** [comment]
Cheers! We'll work that into the repo properly at a later stage
